src/table/body-cell/styles.scss

```diff
@@ -147,28 +148,21 @@ $cell-negative-space-vertical: 2px;
   }
   &:first-child {
     border-inline-start: $border-placeholder;
+    @include cell-padding-inline-start($cell-edge-horizontal-padding);
   }
   &:last-child {
     border-inline-end: $border-placeholder;
     @include cell-padding-inline-end($cell-edge-horizontal-padding);
   }
+  // Using very small padding for 1st column cells in VR.
   &.is-visual-refresh:first-child {
-    &:not(.has-striped-rows) {
-      @include cell-padding-inline-start(awsui.$space-xxxs);
-      &:not(.body-cell-edit-active).body-cell-editable:hover {
-        @include cell-padding-inline-start(calc(#{awsui.$space-xxxs} + #{awsui.$border-divider-list-width}));
-      }
-    }
-
-    &.sticky-cell-pad-inline-start:not(.has-selection) {
-      @include cell-padding-inline-start($cell-horizontal-padding);
+    @include cell-padding-inline-start(awsui.$space-xxxs);
+    &.body-cell-editable:hover {
```

Comment:
Removed the ":not(.body-cell-edit-active)" prefix because the active edit cell cannot be hovered anyways. This also fixes the bug when the wrong padding was assigned for sticky + editable + hovered cells.